projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_financial-chart.scss

```diff
@@ -0,0 +1,159 @@
+/// @param {Number} $x-axis-tick-length [null] - Sets the length of tickmarks along the X-axis.
+/// @param {Color} $x-axis-tick-stroke [null] - Sets the color to apply to tickmarks along the X-axis.
+/// @param {Number} $x-axis-tick-stroke-thickness [null] - Sets the thickness to apply to tickmarks along the X-axis.
+/// @param {String} $x-axis-title-alignment [null] - Sets the horizontal alignment of the X-axis title.
+/// @param {List} $x-axis-title-margin [null] - Sets the margin (top, right, bottom, left) of a title on the X-axis.
+/// @param {Color} $x-axis-title-text-color [null] - Sets the color of the title on the X-axis.
+/// @param {String} $x-axis-title-text-style [null] - Sets the CSS font property for the title on the X-axis.
+/// @param {List} $y-axis-label-margin [null] - Sets the margin (top, right, bottom, left) of labels on the Y-axis.
+/// @param {Color} $y-axis-label-text-color [null] - Sets the color of labels on the Y-axis.
+/// @param {String} $y-axis-label-text-style [null] - Sets the CSS font property for labels on Y-axis.
+/// @param {String} $y-axis-label-vertical-alignment [null] - Sets the vertical alignment of Y-axis labels.
+/// @param {Color} $y-axis-major-stroke [null] - Sets the color to apply to major gridlines along the y-axis.
+/// @param {Number} $y-axis-major-stroke-thickness [null] - Sets the thickness to apply to major gridlines along the Y-axis.
+/// @param {Color} $y-axis-minor-stroke [null] - Sets the color to apply to minor gridlines along the Y-axis.
+/// @param {Number} $y-axis-minor-stroke-thickness [null] - Sets the thickness to apply to minor gridlines along the Y-axis.
+/// @param {Color} $y-axis-strip [null] - Sets the color to apply to stripes along the Y-axis.
+/// @param {Color} $y-axis-stroke [null] - Sets the color to apply to the Y-axis line.
+/// @param {Number} $y-axis-stroke-thickness [null] - Sets the thickness to apply to the Y-axis line.
+/// @param {String} $y-axis-title-alignment [null] - Sets the horizontal alignment of the Y-axis title.
+/// @param {List} $y-axis-title-margin [null] - Sets the margin (top, right, bottom, left) of a title on the Y-axis.
+/// @param {Color} $y-axis-title-text-color [null] - Sets the color of a title on the Y-axis.
+/// @param {String} $y-axis-title-text-style [null] - Sets the CSS font property for the title on the Y-axis.
+/// @param {Number} $y-axis-tick-length [null] - Sets the length of tickmarks along the Y-axis.
+/// @param {Color} $y-axis-tick-stroke [null] - Sets the color to apply to tickmarks along the Y-axis.
+/// @param {Number} $y-axis-tick-stroke-thickness [null] - Sets the thickness to apply to tickmarks along the Y-axis.
+/// @param {String} $zoom-slider-x-axis-major-stroke [null] - Gets or sets stroke brush of major gridlines on x-axis of the zoom slider pane.
+/// @param {Number} $zoom-slider-x-axis-major-stroke-thickness [null] - Gets or sets thickness of major gridlines on x-axis of the zoom slider pane.
```

Comment:
These should be listed as `@prop` and without the `$` sign. Also, all `*-text-style` props don't make sense here, since they should all be set in the typography mixin.